index.js

```diff
@@ -0,0 +1,4 @@
+aaa = 111;
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'aaa' is not defined  no-undef